### PR TITLE
Added a session unit test

### PR DESCRIPTION
**Pull request by codethejason**
The session unit test checks that the session titles are in the correct order as shown on the site.  

scrn

### CODE REVIEW

File: tests/sessionsSpec.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                "Wie Privat ist zu Privat? – Die tägliche Entscheidung eines Familien-Bloggers",
+                "Digitaler Journalismus: Vom Innovationsgeist zur Aufbruchsstimmung"],
+      
+                ["Fashion + Tech Daily Review Day 3",
+                "The Dark Side of Healthcare Innovation: 5 Uneasy Questions We Need to Ask Now",
+                "Wider die Bewilligungskultur im Netz",
+                "Fremd gehen immer nur die anderen – Liebe und Beziehung in Zeiten der Digitalität",
+                "Are we finally becoming human?",
+                "Learn Do Share – Open Design Game",
+                "Unblinding Europe: Klinische Studiendaten ans Licht!",
```
Comment (codethejason) [inline tests/sessionsSpec.js]: The text requires single quotes.

File: tests/sessionsSpec.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    ApiJsonFactory = _ApiJsonFactory_;
+    scope = $rootScope.$new();
+    ctrlLoader = function () {
+      $controller("SessionsController", {
+        $scope: scope,
+        $mdDialog: $mdDialog
+      });
+    };
+    });
+  }));
```
Comment (codethejason) [inline tests/sessionsSpec.js]: There does not seem to be any syntax errors.

File: assets/css/main.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  background: #FFEB00 !important;
+  color: #000; }
+
+.md-card-content {
+  background: #FFEB00 !important;
+  color: #000; }
+
+.md-subheader-content {
+  color: #000 !important;
+  border-color: !important; }
```
Comment (juslee) [inline assets/css/main.css]: Codacy Issue found: [Unexpected token '!important' at line 15, col 17.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579142/issues/source?bid=3233857&fileBranchId=3233877#l15)

File: assets/css/main.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
+.md-toolbar-tools {
+  background: #FFEB00 !important;
```
Comment (juslee) [inline assets/css/main.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579142/issues/source?bid=3233857&fileBranchId=3233877#l2)

File: assets/css/main.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  color: #000; }
+
+.md-subheader-content {
+  color: #000 !important;
+  border-color: !important; }
+
+.md-input-focused:not(.md-input-invalid) > input {
+  color: #000 !important;
+  border-color: #000 !important; }
```
Comment (juslee) [inline assets/css/main.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579142/issues/source?bid=3233857&fileBranchId=3233877#l19)

File: assets/css/main.css [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+.md-grid-tile-footer {
+  background: #FFEB00 !important;
+  color: #000; }
+
+.md-card-content {
+  background: #FFEB00 !important;
+  color: #000; }
+
+.md-subheader-content {
+  color: #000 !important;
```
Comment (juslee) [inline assets/css/main.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579142/issues/source?bid=3233857&fileBranchId=3233877#l14)

File: karma.conf.js [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+      'app/components/**/*.js': 'coverage'
+    },
+
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ['progress'],
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579158/issues/source?bid=3233857&fileBranchId=3233877#l44)

File: karma.conf.js [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      'assets/js/leaflet.js',
+      'bower_components/angular/angular.js',
+      'node_modules/angular-mocks/angular-mocks.js',
+      'bower_components/ngstorage/ngStorage.js',
+      'bower_components/angular-aria/angular-aria.js',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579158/issues/source?bid=3233857&fileBranchId=3233877#l18)

File: karma.conf.js [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+      'bower_components/angular-material/angular-material.js',
+      'bower_components/angular-animate/angular-animate.js',
+      'bower_components/angular-ui-router/release/angular-ui-router.js',
+      'app.js',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579158/issues/source?bid=3233857&fileBranchId=3233877#l22)

File: karma.conf.js [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      'assets/js/leaflet.js',
+      'bower_components/angular/angular.js',
+      'node_modules/angular-mocks/angular-mocks.js',
+      'bower_components/ngstorage/ngStorage.js',
+      'bower_components/angular-aria/angular-aria.js',
+      'bower_components/angular-material/angular-material.js',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579158/issues/source?bid=3233857&fileBranchId=3233877#l19)

File: karma.conf.js [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+    browsers: ['Chrome'],
+
+
+    // Continuous Integration mode
+    // if true, Karma captures browsers, runs the tests and exits
+    singleRun: false,
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579158/issues/source?bid=3233857&fileBranchId=3233877#l75)

File: karma.conf.js [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      'assets/js/leaflet.js',
```
Comment (juslee) [inline karma.conf.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406579158/issues/source?bid=3233857&fileBranchId=3233877#l14)

**Pull request closed without merging**